docs/basics/beginners-contributing-guide.md

```diff
@@ -0,0 +1,119 @@
+# Beginners Contributing Guide
+
+## Prerequisites:
+* Docker is already running in your environment
+* You have a [Github](https://github.com) account and are logged in
+* This guide was written for linux
+
+__Note__: **This document will use variables thought out make sure to change them as needed.**
+
+### 1. Install Git
+This section will install Git.
+
+0. Log in to the server you are going to use for Git.
+
+0. Type `add-apt-repository ppa:git-core/ppa`.
```

Comment:
You can just install git from the repos, don't need to add a ppa and pollute your installation.

Reply:
done